static/js/common.js

```diff
@@ -134,3 +135,51 @@ export function adjust_mac_shortcuts(key_elem_class, require_cmd_style) {
         $(this).text(key_text);
     });
 }
+
+// See https://zulip.readthedocs.io/en/latest/development/authentication.html#password-form-implementation
+// for design details on this feature.
+function set_password_toggle_label(password_selector, label, tippy_tooltips) {
+    $(password_selector).attr("aria-label", label);
+    if (tippy_tooltips) {
+        const tippy_selectors = tippy(password_selector);
```

Comment:
`tippy_instances` is better variable name as it is commonly used in the tippy docs. Also, use `instance` below.